README.md

```diff
@@ -11,11 +11,15 @@ GraphSpace has three dummy users:
 
 Requirements
 ===================================
-1. Python v2.7.10
+
+1. [Python](https://askubuntu.com/a/101595) v2.7.10
 2. [postgreSQL](https://github.com/Murali-group/GraphSpace/wiki/PostgreSQL-Installation) with pg_trgm extension
-3. virtualenv
-4. [bower](https://bower.io/)
-5. [ElasticSearch](https://github.com/Murali-group/GraphSpace/wiki/Steps-for-setting-up-ElasticSearch-on-AWS)
+3. [virtualenv](https://virtualenv.pypa.io/en/stable/installation/)
+4. [bower](https://bower.io/) _dependant on [Node.js](https://github.com/Murali-group/GraphSpace/wiki/Install-Node.js)_
+5. [Apache Kafka](https://github.com/Murali-group/GraphSpace/wiki/Install-Apache-Kafka)
+6. [ElasticSearch](https://github.com/Murali-group/GraphSpace/wiki/Steps-for-setting-up-ElasticSearch-on-AWS)
+7. [Redis](https://github.com/Murali-group/GraphSpace/wiki/Install-Redis)
+8. [Supervisor](http://supervisord.org/installing.html#installing-via-pip)
```

Comment:
@melvin15may Don't worry about adding the explanations to the README. However, could you add the "create" commands above to the Postgres installation wiki page?